packages/libtiff.rb

```diff
@@ -22,23 +22,24 @@ class Libtiff < Package
      x86_64: '33129073011fd853518e91376a84fa0ddbc61605c041297c0d130294a113f389'
   })
 
```

Comment:
Update to https://download.osgeo.org/libtiff/tiff-4.5.0.tar.xz?

Reply:
Building 4.5.0...